discord-bot/bot.js

```diff
@@ -48,61 +51,69 @@ bot.on('message',message => {
 
 bot.on('message',message => {
 if(message.content === '!xkcd')
-{ 
+   { 
     let comicNo = Math.floor(Math.random() * (2000 - 100 + 1) + 100)
     
     axios.get(`http://xkcd.com/${comicNo}/info.0.json`).then(resp => {
     
         message.channel.send(resp.data.img)
     });
-    
-    }
+   }
     
 })
 
 //Social media commands
 
 bot.on('message',message => { 
+
    if(message.content === '!website')
    {    
     const website = new Discord.MessageEmbed().setTitle('Visit our Website').setURL('https://osdc.netlify.app/')
     message.channel.send(website)
+    return false;
    }
-   if(message.content === '!twitter')
+   else if(message.content === '!twitter')
    {
         const twitter =  new Discord.MessageEmbed().setTitle('Check us out on Twitter').setURL('https://twitter.com/osdcjiit')
         message.channel.send(twitter)
+        return false
    }
-   if(message.content === '!facebook')
+   else  if(message.content === '!facebook')
    {
         const facebook = new Discord.MessageEmbed().setTitle('Follow us on Facebook').setURL('https://www.facebook.com/JIIT-OSDC-169171359799320/')
         message.channel.send(facebook)
    }
-    if(message.content === '!github')
+   else  if(message.content === '!github')
     {
-        const github = new Discord.MessageEmbed().setTitle('Take a look at our cool projects').setURL('https://github.com/osdc')
+        const github = new Discord.MessageEmbed().setTitle(' Take a look at our cool projects ').setURL('https://github.com/osdc')
         message.channel.send(github)
     }
-    if(message.content === '!telegram')
+    else  if(message.content === '!telegram')
     {
         const telegram = new Discord.MessageEmbed().setTitle(' Join our Telegram Channel').setURL('https://t.me/jiitosdc')
         message.channel.send(telegram)
     } 
-    if(message.content === '!irc')
+    else  if(message.content === '!irc')
     {
         const irc = new Discord.MessageEmbed().setTitle(' Join us on IRC server of Freenode at #jiit-lug').setURL('https://github.com/osdc/community-committee/wiki/IRC')
         message.channel.send(irc)
     } 
-    if(message.content === '!blog')
+    else  if(message.content === '!blog')
     {
         const blog = new Discord.MessageEmbed().setTitle('Blogs written by the folks at the Open Source Developers Community').setURL('https://osdcblog.netlify.com/')
         message.channel.send(blog)
     } 
     
-    if(message.content === '!instagram')
+    else  if(message.content === '!instagram')
     {
         message.channel.send('https://tenor.com/view/dont-do-that-avengers-black-panther-we-dont-do-that-here-gif-12042935')
     }
+
+    else if(message.content.length > 0 && isValid.has(message.content) === false && message.content[0] === '!')
```

Comment:
Is the `isValid` needed? This case will only execute if there are no matches in any of the above conditions :thinking:

Reply:
Earlier i though of implementing it in a different way and hence that` isValid ` , i forgot to remove it :|

Reply:
Updated some part of the code , let me know if there is any discrepancy left :)